walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js

```diff
@@ -15,6 +15,13 @@
 /**
  * Fetches a random quote from the server and adds it to the DOM.
  */
+
+function getGreeting(){
+    fetch('/greetings').then(response => response.text()).then((greeting) => {
+    document.getElementById('greeting-container').innerText = greeting;
+    });
+}
+
```

Comment:
Please don't check in changes to the "walkthroughs" code.

Reply:
Okay